packages/cli/src/generators/app.ts

```diff
@@ -64,7 +68,7 @@ class AppGenerator extends Generator<AppGeneratorOptions> {
 
     const result = spawn.sync(this.options.yarn ? 'yarn' : 'npm', ['install'], {stdio: 'inherit'})
     if (result.status !== 0) {
-      throw new Error()
+      throw new Error(`Failed to ${this.options.yarn ? 'yarn' : 'npm'} install`)
     }
 
     console.log(chalk.hex(themeColor).bold('\nDependencies successfully installed.'))
```

Comment:
Curious why we are using `log` helper and/or `console.log|console.error`. Should it be unified?

Reply:
I've been incrementally migrating to the `@blitzjs/server/utils/log` helper as I touch relelvant lines. We haven't touched the app generator meaningfully in a bit so I haven't had a chance to migrate these log lines. If you want to migrate these to use the main logging utility it would be a huge help!!

Reply:
Done https://github.com/blitz-js/blitz/pull/367/commits/c31ba558e752cbeeba225dd0c73b8cb5d01c25b4